# Center for Proteomics and Metabolomics

## Central Code Hub

This is a hierarchic list of projects, packages, workflows and code snippets, developed or frequently used at the CPM, as links to latest or most recent stable releases or repos in the organization:

- Data I/O
- Calibration and Alignment
  - [mzrecal](https://github.com/524D/mzrecal): calibrates mzML MS1 data using identified peptides
  - [msalign](http://www.ms-utils.org/msalign/index.html): chromatographically aligns LC-MS/MS datasets ("MBR")
- Identification
  - [Refactored-FastPaSS](https://github.com/magnuspalmblad/Refactored-FastPaSS): searches spectral libraries using GPUs
- Mass Spectrometry Imaging
  - [addFeatures](https://github.com/magnuspalmblad/addFeatures): adds spot images to a SCiLS Lab dataset from a list of defined mass spectral features
  - [averageRegions](https://github.com/magnuspalmblad/averageRegions): averages the spectra in defined regions in a SCiLS Lab dataset
- Quantitation
  - [Xtractor](http://ms-utils.org/Xtractor/): extracts areas under predefined m/z regions
  - [LaCyTools](https://git.lumc.nl/cpm/lacytools): integrates LC-MS peaks (and more)
- Statistics
  - [MSQC](https://github.com/Center-for-Proteomics-and-Metabolomics/MSQC): R Markdowns for calculating and visualizing QC metrics for mass spectrometry
- Visualization
  - [SCOPE](https://github.com/ReinV/SCOPE): visualizes properties of molecules in literature search
- Other
  - [compareMS2](https://github.com/524D/compareMS2): calculates the global similarity between tandem mass spectrometry datasets (and more)
  - [GlycoDash](https://github.com/Center-for-Proteomics-and-Metabolomics/GlycoDash): R Shiny dashboard for processing glycomics data
  - [NGP](https://github.com/magnuspalmblad/NGP): simulation of single-molecule peptide sequencing
  - [shic](https://github.com/magnuspalmblad/shic): collection of shims for workflow composition
